src/rdo.rs

```diff
@@ -1007,7 +1008,7 @@ pub fn rdo_tx_type_decision<T: Pixel>(
         CFLParams::default(), // Unused.
         true,
         rdo_type,
-        true
```

Comment:
This was the problem, i.e. it has been always true making inverse_tx skipped for some cases.